src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,4 @@
+@mixin size ($size) {
+  height: $size;
+  min-width: $size;
```

Comment:
It is enough for you to indicate only the card height, because you set width in grid-template-columns